packages/CodeGenLib/src/Database/sql_codegen.ts

```diff
@@ -799,7 +799,8 @@ ${whereClause}GO${permissions}
     protected generateSPCreate(entity: EntityInfo): string {
         const spName: string = entity.spCreate ? entity.spCreate : `spCreate${entity.ClassName}`;
         const firstKey = entity.FirstPrimaryKey;
-        const primaryKeyAutomatic: boolean = firstKey.AutoIncrement || (firstKey.Type.toLowerCase().trim() === 'uniqueidentifier' && firstKey.DefaultValue && firstKey.DefaultValue.trim().length > 0);
+        const IsPrimaryKey = (firstKey.Type.toLowerCase().trim() === 'uniqueidentifier') && (firstKey.DefaultValue && firstKey.DefaultValue.trim().length > 0);
```

Comment:
@JS-BC  I see you are trying to make this more readable. However your new variable name IsPrimaryKey isn't what the calculation reflects, it is calculation if the type is unique and the defaultvalue exists which relates to the earlier name primaryKeyAutomatic. I don't like this separation for that reason because IsPrimaryKey is an incorrect variable name, just go back to what we had before or use a variable name that reflects what the variable actually holds